docker-compose.yml

```diff
@@ -71,6 +71,19 @@ services:
       POSTGRES_PASSWORD: ${POSTGRES_PASSWORD:-c2VjcmV0Cg==}
       POSTGRES_USER: ${POSTGRES_USER:-postgres}
       POSTGRES_HOST: ${POSTGRES_HOST:-postgres}
+  
+  localstack:
```

Comment:
Including localstack with iam service for integration testing. Will be able to calibrate more when the DDL is enabled in the image.